docs/alerts.rst

```diff
@@ -203,6 +203,31 @@ The full list of all the attributes can be found in the
                      ttp: ['PORTSCAN', 'MITRE_T1046_NETWORK_SERVICE_SCANN...
     ...
 
+Event Details
+^^^^^^^^^^^^^
+
+You can retrieve the event(s) behind a ``CBAnalyticsAlert`` easily with its ``get_events()`` method::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.platform import CBAnalyticsAlert
+    >>> api = CBCloudAPI(profile='platform')
+    >>> query = cb.select(CBAnalyticsAlert).set_create_time(range="-4w")
+    >>> # get the first alert returned by the query
+    >>> alert = query[0]
+    >>> for event in alert.get_events():
```

Comment:
How can we make this line standout more as we are trying to call out `get_events` and it gets lost among  the other lines. I recognize this is a pattern we have not applied any where else but does bolding that function make sense? is that enough to make it stand out?

Reply:
Maybe add an extra comment on the line above? I don't think you can apply bolding to preformatted text.

Reply:
Open to options as to what would draw attention to that line in particular. Would be great to define a pattern we can re-use for other examples

Reply:
I'm adding the extra comment on the line above, which, as I said, is basically all we _can_ do in a preformatted example.

Reply:
What do you think on how well it actually makes it stand out? or does it just add more text?

Reply:
Good enough for now.  The Alerts guide will have significant updates with the v7 changes in SDK 1.5.0.  Likely will become getting observations for an Alert instead of events.